docs/management/advanced-options.asciidoc

```diff
@@ -393,6 +393,26 @@ value is 10000.
 [[apm-enable-service-overview]]`apm:enableServiceOverview`::
 When enabled, displays the *Overview* tab for services in *APM*.
 
+[[apm-agent-explorer]]`observability:apmAgentExplorerView`::
+beta:[] Enables the Agent explorer view.
+
+[[apm-aws-price]]`observability:apmAWSLambdaPriceFactor`::
+Set the price per Gb-second for your AWS Lambda functions.
+
+[[apm-aws-request]]`observability:apmAWSLambdaRequestCostPerMillion`::
+??
```

Comment:
@cauemarcondes what does this config do? I couldn't figure it out.

Reply:
It's used to calculate the cost running on serverless:

https://github.com/elastic/kibana/blob/5fca22a79db0ad72d539d7ee297211653e607026/x-pack/plugins/apm/server/routes/metrics/serverless/helper.ts#L90-L92

The price is the number of requests (throughput) multipled with the AWS Lambda request cost (currently the monthly request price is $0.20 per one million request)